concent_api/core/admin.py

```diff
@@ -16,9 +27,53 @@
 from .models import StoredMessage
 from .models import Subtask
 from .models import SubtaskWithTimingColumnsManager
+from .payments.report import create_report
 
 ACTIVE_STATE_NAMES = [x.name for x in Subtask.ACTIVE_STATES]
 PASSIVE_STATE_NAMES = [x.name for x in Subtask.PASSIVE_STATES]
+DECORATORS = [transaction.non_atomic_requests(using='control')]
+
+
+@method_decorator(DECORATORS, name='dispatch')
+class PaymentReportAdminView(TemplateView):
+
+    template_name = 'admin/payment_report.html'
+
+    def get_context_data(self, **kwargs: Any) -> Dict:
+        context_data = super().get_context_data(**kwargs)
+        context_data['payment_report'] = create_report()
+        context_data['eth_network'] = self._get_eth_network()
+        return context_data
+
+    @staticmethod
+    def _get_eth_network() -> str:
+        session = requests.Session()
+        data = json.dumps({'id': 67, 'jsonrpc': '2.0', 'method': 'net_version', 'params': []})
+        headers = {'Content-type': 'application/json'}
+        try:
+            response = session.post(settings.GETH_ADDRESS, data=data, headers=headers)  # type: ignore
+        except ConnectionError:
+            return ''
```

Comment:
Maybe return `couldn't connect` or something like this?

Reply:
I don't know if we want to crush concent just because GETH from settings is not working at that moment. Do we want to render payment report without payments? Or we could render custom error 5xx page, that shows exceptions?

Reply:
I am not talking about crashing, I am talking about returning and showing a string with info that Ethereum network could not be retrieved.

Reply:
As we spoke, this string is a part of link in template, so it need to stay like that. Or we should change handling that exception completely, @cameel ?

Reply:
We do not need to contact Geth to get this information. We should already know if we're using mainnet or testnet. We need this information to initialize SCI anyway.

BTW, I have noticed that we always tell SCI to use Rinkeby, no matter whether we're using mainnet or testnet. That's a bug: #1120.

Reply:
I didn't find any other way. There is no setting that would define that right now. `GETH_ADDRESS` is usually just some ip address. It's going to be much easier once #1120 is done.

Reply:
You can just hard-code it to Rinkeby and add a note in #1120 to use the setting for both. Right now Concent is hard-coded for Rinkeby anyway.